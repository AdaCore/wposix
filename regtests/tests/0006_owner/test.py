from test_support import *

gnatmake('owner');
run('owner')
