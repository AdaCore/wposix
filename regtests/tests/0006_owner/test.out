OK, F/P UID
OK, F/P GID
OK F_UID seems correct.
OK F_GID seems correct.
OK P_UID seems correct.
OK P_GID seems correct.
